cothink2/client/item.js

```diff
@@ -0,0 +1,16 @@
+Item = {};
```

Comment:
Meteor fixes that. Variables without declaration become global (app-wide) variables. Declared variables are scoped to their file.